nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -338,8 +338,10 @@ in
 
       # Enable helpful DBus services.
       services.accounts-daemon.enable = true;
+      programs.dconf.enable = true;
       # when changing an account picture the accounts-daemon reads a temporary file containing the image which systemsettings5 may place under /tmp
       systemd.services.accounts-daemon.serviceConfig.PrivateTmp = false;
+      services.gnome.glib-networking.enable = true;
```

Comment:
I think every program that is not working without this is missing something in its wrapper.